pwndbg/commands/version.py

```diff
@@ -189,7 +195,20 @@ def bugreport(run_browser=False):
     github_issue_url = "https://github.com/pwndbg/pwndbg/issues/new"
     github_issue_body = "?body=" + quote(issue_bugreport)
 
-    if run_browser:
+    if use_gh:
+        try:
+            with NamedTemporaryFile("w", delete=True) as f:
+                f.write(issue_bugreport)
+                f.flush()
+                if "EDITOR" in os.environ:
+                    editor = os.environ["EDITOR"]
+                else:
+                    editor = "vi"
```

Comment:
You can simplify this with either `os.environ.get("EDITOR", "vi")` or `os.getenv("EDITOR", "vi")`.

Reply:
And this isn't necessary for this PR, but in the future we should probably fallback to a config option instead of hardcoding "vi" here.